Tasks/AzureAppServiceManageV0/azureappservicemanage.ts

```diff
@@ -43,10 +49,23 @@ async function run() {
         if(action != "Swap Slots" && !slotName) {
             resourceGroupName = await AzureResourceFilterUtils.getResourceGroupName(azureEndpoint, 'Microsoft.Web/Sites', webAppName);
         }
+        if(action == "Complete Swap")
+        {
```

Comment:
can we have same key for different values in picklist for action

Reply:
Just checked. We cannot have the same key for different values.

Reply:
instead of changing the action, you can have like
case "Swap Slots":
case "Complete Swap": 
{}